src/order-history/saga.js

```diff
@@ -1,11 +1,13 @@
 import { call, put, takeEvery } from 'redux-saga/effects';
+import { push } from 'connected-react-router';
+import LoggingService from '@edx/frontend-logging';
 
 // Actions
 import {
   FETCH_ORDERS,
   fetchOrdersBegin,
   fetchOrdersSuccess,
-  fetchOrdersFailure,
+  // fetchOrdersFailure,
```

Comment:
Remove this and the method's definition.

Reply:
👍